broadinstitute/pearl
# JN-533 submitted surveys go back to first page

On submission events, this resets the page number to 1, so that if the user returns, they'll go back to the beginning.  

TO TEST:
1. sign into participant view as jsalk@test.com
2. fill out an uncompleted survey
3. hit submit
4. from the dashboard, go back to that survey
5. confirm you get taken back tot he first page
